spikecomparison/multisortingcomparison.py

```diff
@@ -10,31 +12,120 @@
 
 class MultiSortingComparison(BaseComparison):
     def __init__(self, sorting_list, name_list=None, delta_time=0.4, sampling_frequency=None,
-                 match_score=0.5, chance_score=0.1, n_jobs=-1, verbose=False):
-
+                 match_score=0.5, chance_score=0.1, n_jobs=-1, spiketrain_mode='union', verbose=False,
+                 do_matching=True):
         BaseComparison.__init__(self, sorting_list, name_list=name_list,
                                 delta_time=delta_time, sampling_frequency=sampling_frequency,
                                 match_score=match_score, chance_score=chance_score,
                                 n_jobs=n_jobs, verbose=verbose)
-        self._do_matching(verbose)
+        self._spiketrain_mode = spiketrain_mode
+        if do_matching:
```

Comment:
what is the usecase of do_mathcing=False ?

Reply:
when loading the graph from file